doc/pages/example_notebooks/density/densitys_easy_example.ipynb

```diff
@@ -184,16 +199,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## customize a density with different density models\n",
+    "## Customize a density with different density models.\n",
     "To customize a density use the DensityList.\n",
-    "In a superposition of globel models of density distribution of the Milkyway keep care of normalisation. Therfore you can just add components by deactivating the other  "
+    "In a superposition of global models of the density distribution of the Milky Way take care of the normalisation. For this purpose you can just add components after deactivating the unwanted parts.  "
```

Comment:
This sentence does not have a subject. I am also not sure what we want to say here. 
Does it mean that the user has to calculate the normalization on their own?

Reply:
I've changed this sentence in the last commit. I think it means that the script given there shows how to take into account the normalization properly for a superposition of global models.